docs/release_notes/ibexa_dxp_v5.0.md

```diff
@@ -0,0 +1,335 @@
+    - `\Ibexa\Contracts\ConnectorAi\ActionConfiguration\Event\CreateActionConfigurationEvent`
+    - `\Ibexa\Contracts\ConnectorAi\ActionConfiguration\Event\BeforeUpdateActionConfigurationEvent`
+    - `\Ibexa\Contracts\ConnectorAi\ActionConfiguration\Event\UpdateActionConfigurationEvent`
+    - `\Ibexa\Contracts\ConnectorAi\ActionConfiguration\Event\BeforeDeleteActionConfigurationEvent`
+    - `\Ibexa\Contracts\ConnectorAi\ActionConfiguration\Event\DeleteActionConfigurationEvent`
+    - `Ibexa\Contracts\ConnectorAi\Events\ContextEvent`
+    - `Ibexa\Contracts\ConnectorAi\Events\ResolveActionConfigurationWidgetConfigEvent`
+    - `Ibexa\Contracts\ConnectorAi\Events\ResolveActionHandlerEvent`
+
+- ibexa/discounts
+
+    - `\Ibexa\Contracts\Discounts\Event\BeforeCreateDiscountEvent`
+    - `\Ibexa\Contracts\Discounts\Event\CreateDiscountEvent`
+    - `\Ibexa\Contracts\Discounts\Event\BeforeDeleteDiscountEvent`
+    - `\Ibexa\Contracts\Discounts\Event\DeleteDiscountEvent`
+    - `\Ibexa\Contracts\Discounts\Event\BeforeUpdateDiscountEvent`
+    - `\Ibexa\Contracts\Discounts\Event\UpdateDiscountEvent`
+    - `\Ibexa\Contracts\Discounts\Event\CreateDiscountCreateStructEvent`
+    - `\Ibexa\Contracts\Discounts\Event\CreateDiscountUpdateStructEvent`
+    - `\Ibexa\Contracts\Discounts\Event\CreateFormDataEvent`
+    - `\Ibexa\Contracts\Discounts\Event\MapDiscountToFormDataEvent`
+    - `\Ibexa\Contracts\Discounts\Event\Step\CreateFormDataEvent`
+    - `\Ibexa\Contracts\Discounts\Event\Step\MapCreateDataToStructEvent`
+    - `\Ibexa\Contracts\Discounts\Event\Step\MapDiscountToFormDataEvent`
+    - `\Ibexa\Contracts\Discounts\Event\Step\MapUpdateDataToStructEvent`
+    - `\Ibexa\Contracts\Discounts\Admin\Form\Event\PreDiscountCreateEvent`
+    - `\Ibexa\Contracts\Discounts\Admin\Form\Event`
+    - `\Ibexa\Contracts\DiscountsCodes\Event\BeforeDiscountCodeApplyEvent`
+
+#### Twig functions
+
+The following Twig functions have been added in the v5.0 release:
+
+- `ibexa_ai_config`
+- `ibexa_render_discount_rule_type`
+- `ibexa_discounts_render_discount_badge`
+- `ibexa_get_original_price`
+- `ibexa_format_discount_value`
+- `ibexa_discounts_is_active`
+- `ibexa_discounts_form_themes`
+- `ibexa_discounts_can_edit`
+- `ibexa_discounts_can_enable`
+- `ibexa_discounts_can_disable`
+- `ibexa_discounts_can_delete`
+
+#### Other upgrades
+
+This release brings other minor upgrades intended to improve the developer's experience:
+
+- New icons that serve as a highlight of a future system design
+- Improved DX with capabilities offered by PHP in version 8.3. For example, the `AsTwigComponent` attribute [facilitates autoconfiguration](https://github.com/ibexa/twig-components/pull/15) of Twig components
+- The introduction of [[= product_name_base =]] Rector package that is based on [Rector](https://github.com/rectorphp) and comes with additional rules for working with Ibexa code. You can use it to get rid of PHP code deprecations.
```

Comment:
Linking to the original rector is good, let's link to https://github.com/ibexa/rector as well